File: solutions/csharp/log-levels/1/LogLevels.cs
static class LogLine
{
    public static string Message(string logLine)
    {
        return logLine.Split(":")[1].Trim();
    }

    public static string LogLevel(string logLine)
    {
       return logLine.Split(":")[0].Replace("[", "").Replace("]", "").ToLower();
    }

    public static string Reformat(string logLine)
    {
        string logLevel = LogLine.LogLevel(logLine);
        string message = LogLine.Message(logLine);

        return $"{message} ({logLevel})";
    }
}

File: solutions/java/log-levels/1/src/main/java/LogLevels.java
public class LogLevels {
    
    public static String message(String logLine) {
        return logLine.split(":")[1].trim();
    }

    public static String logLevel(String logLine) {
        return logLine.split(":")[0].replace("[", "").replace("]","").toLowerCase();
    }

    public static String reformat(String logLine) {
        String message = LogLevels.message(logLine);
        String loglevel = LogLevels.logLevel(logLine);

        return message + " " + "(" + loglevel + ")";
    }
}
